clients/python/llmengine/model.py

```diff
@@ -41,10 +41,10 @@ def create(
         quantize: Optional[Quantization] = None,
         checkpoint_path: Optional[str] = None,
         # General endpoint fields
-        cpus: int = 32,
-        memory: str = "192Gi",
+        cpus: int = 8,
+        memory: str = "40Gi",
```

Comment:
@gargutsav @auag92 i'm updating these default values, so if you're still going with 4 GPUs, please make sure values are set same as how you're using them right now